patches/minecraft/net/minecraft/server/Main.java.patch

```diff
@@ -1,11 +1,21 @@
 --- a/net/minecraft/server/Main.java
 +++ b/net/minecraft/server/Main.java
-@@ -76,6 +_,8 @@
+@@ -76,6 +_,18 @@
        OptionSpec<String> optionspec12 = optionparser.accepts("serverId").withRequiredArg();
        OptionSpec<Void> optionspec13 = optionparser.accepts("jfrProfile");
        OptionSpec<String> optionspec14 = optionparser.nonOptions();
 +      optionparser.accepts("allowUpdates").withRequiredArg().ofType(Boolean.class).defaultsTo(Boolean.TRUE); // Forge: allow mod updates to proceed
 +      optionparser.accepts("gameDir").withRequiredArg().ofType(File.class).defaultsTo(new File(".")); //Forge: Consume this argument, we use it in the launcher, and the client side.
++      OptionSpec<net.minecraft.core.BlockPos> spawnPosOpt;
++      OptionSpec<String> enableNamespaceOpt;
++      boolean gametestEnabled = net.minecraftforge.gametest.ForgeGameTestHooks.isGametestEnabled();
```

Comment:
While these are effectively final because they're used in a lambda, I'd like them to be marked `final` to clarify that.